roles/logs_client/tasks/firewall.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Ensure that the directory iptables extras exists
+  ansible.builtin.file:
+    path: "{{ iptables_extras_dir }}"
+    owner: "root"
+    group: "root"
+    mode: '0770'
+    state: 'directory'
+  delegate_to: "{{ item }}"
+  register: create_iptables_extras_d
+  loop: "{{ rsyslogs_ext_ips }}"
+  become: true
+
+- name: Get public IP of all the clients
+  ansible.builtin.uri:
+    url: http://ifconfig.me/ip
+    return_content: true
+  register: client_public_ip
+  changed_when: false
+
+- name: Print public IP of all the clients
+  ansible.builtin.debug:
+    msg: "{{ inventory_hostname }} => {{ client_public_ip.content }}"
+  changed_when: false
+
+- name: Add line to Logs exception file
+  ansible.builtin.lineinfile:
+    path: '{{ iptables_extras_dir }}/{{ stack_prefix }}.allow'
+    state: present
+    mode: '0640'
+    create: true
+    regexp: "{{ client_public_ip.stdout | trim }} 41514"
+    line: "{{ client_public_ip.stdout | trim }} 41514"
+  run_once: true
+  delegate_to: "{{ item }}"
+  register: add_ip_to_firewall_extra_allow
+  loop: "{{ rsyslogs_ext_ips }}"
+  become: true
+  notify: restart_rserver_firewall
```

Comment:
Confusing handler name: I would expect something with syslog in the name too. "rserver" alone suggests this is about `R server`, which means running **_R_** (the statistics language interpreter) in server mode.

Reply:
Absolutely true. I noticed it also, and had already renamed it ;)